src/docs/markdown/v2-upgrade.md

```diff
@@ -146,6 +147,46 @@ browse /subfolder/
 file_server /subfolder/* browse
 ```
 
+### errors
+
+Custom error pages can be accomplished with [`handle_errors`](/docs/caddyfile/directives/handle_errors).
+
+
+- **v1:**:
+
+```
+errors {
+	404 404.html # Not Found
+	500 500.html # Internal Server Error
```

Comment:
```suggestion
	404 404.html
	500 500.html
```

The comments don't add anything important here, and are distracting, so let's take them out.